Tasks/SonarQubePreBuild/SonarQubePreBuildImpl.ps1

```diff
@@ -181,4 +180,16 @@ function GetEndpointData
 	}
 
     return $serviceEndpoint
+}
+
+function GetDashboardUrl
```

Comment:
I found a minor bug - if the server url has a trailing /, the report uri will be invalid, so I am removing the trailing slash.